test/unit/css.js

```diff
@@ -203,6 +203,59 @@ test( "css() explicit and relative values", 29, function() {
 	equal( $elem.css("opacity"), "1", "'+=0.5' on opacity (params)" );
 });
 
+test( "css() non-px relative values (gh-1711)", 17, function() {
+	var cssCurrent,
+		units = {},
+		$parent = jQuery("#nothiddendiv"),
+		$child = jQuery("#nothiddendivchild", $parent),
+		add = function( prop, val, unit ) {
+			var str = ( val < 0 ? "-=" : "+=" ) + Math.abs(val) + unit;
+			$child.css( prop, str );
+			equal(
+				Math.round( parseFloat( $child.css( prop ) ) ),
+				Math.round( cssCurrent += val * units[ prop ][ unit ] ),
+				"'" + str + "' on " + prop
```

Comment:
Yeah, I prefer that way, but I did that because the previous unit test `"css() explicit and relative values"` write the debug like: `"'-=9px' on width (hash)"`.
Do you want change this part too? It's fast and easy to do that with multi cursor ^^